scout/constants/case_tags.py

```diff
@@ -3,8 +3,9 @@
 # keys are used in the load report cli, while key_name is saved in case database documents
 CUSTOM_CASE_REPORTS = {
     "delivery": {"key_name": "delivery_report", "format": "HTML", "pdf_export": True},
-    "cnv": {"key_name": "cnv_report", "format": "PDF", "pdf_export": True},
+    "cnv": {"key_name": "cnv_report", "format": "PDF", "pdf_export": False},
     "cov_qc": {"key_name": "coverage_qc_report", "format": "HTML", "pdf_export": True},
+    "exe_ver": {"key_name": "exe_ver", "format": "YAML", "pdf_export": True},
```

Comment:
The downloaded PDF version of this report (when you click on the right side button) doesn't look that good:

<img width="1366" alt="image" src="https://user-images.githubusercontent.com/28093618/203707194-8657bd04-5cb9-4085-a5b8-1359be50fd49.png">

Perhaps just keep the other button and serve the yaml?

Reply:
Very valid point. I changed it to preformatted instead, so this:
<img width="1015" alt="Screenshot 2022-11-24 at 08 22 01" src="https://user-images.githubusercontent.com/758570/203719740-a4efe0c9-dfa4-44e9-bb62-886699f28848.png">

Reply:
Looks great now!